### PR TITLE
Fix minor spec typos

### LINKED ISSUE #179: Needs “O” at end of callbackfn parameters
[JP22] In step 8-c-ii of “22.1.3.19 Array.prototype.reduceRight(callbackfn [, initialValue])”, callbackfn, which takes four arguments, called with three arguments (“«accumulator, kValue, k, »”).  Need “O” at the end of parameters (e.g., «accumulator, kValue, k, O»)?

### PR DESCRIPTION
**Pull request by ljharb**
This fixes #179 and #180.

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -30893,7 +30893,7 @@
           1. Assert: Type(_length_) is not Object.
           1. If NewTarget is *undefined*, throw a *TypeError* exception.
           1. If _length_ is *undefined*, throw a *TypeError* exception.
-          1. Let _numberLength_ be ToNumber(_length_).
+          1. Let _numberLength_ be ? ToNumber(_length_).
```
Comment (anba) [inline spec.html]: Abrupt completions for `ToNumber` are checked by `?ToLength`, so having both, `?ToNumber` and `?ToLength`, is redundant.

Comment (ljharb) [inline spec.html]: aha, that explains why it's not there. I'll excise the commit, thanks.

Comment (anba) [inline spec.html]: It's also okay to move the `?` from ToLength to ToNumber (probably even easier to understand - 
I think most would prefer to remove leading ReturnIfAbrupts in abstract operations and instead check for abrupt completions in the caller.).

**Comment by bterlson** [comment]
Committed as c8b4aee.

**Pull request closed without merging**